orca/orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/config/WebhookProperties.java

```diff
@@ -159,6 +159,12 @@ public static class AllowedRequest {
 
     /** The url must match this pattern to be considered valid */
     private String urlPattern;
+
+    /**
+     * Whether it's safe to retry requests to urls that match. GET requests are always considered
+     * safe, but this enables retries of other methods as well.
+     */
+    private boolean safeToRetry;
```

Comment:
Is this config per webhook, or for all webhooks? Seems like it would be valuable to be able to configure it for specific patterns?